src/pages/migrating/guide/develop.mdx

```diff
@@ -0,0 +1,167 @@
+| CSS Grid                         | <Tag type="cyan">Breaking</Tag>  | [CSS Grid](#css-grid)                               | `@carbon/grid` `Grid` `Row` `Column`                                                                                       |
+| Design tokens                    | <Tag type="cyan">Breaking</Tag>  | [SASS Modules](#sass-modules)                       | All components                                                                                                             |
+| Size prop                        | <Tag type="cyan">Breaking</Tag>  | [Size prop](#size-prop-updates)                     | `Button` `DataTable` `Modal` `ComboBox` `Dropdown` `TextInput` `Search` `DatePicker` `Tag` `Toggle` `Select` `MultiSelect` |
+| UI Shell theming                 | <Tag type="cyan">Breaking</Tag>  | [SASS Modules](#sass-modules)                       | `UI Shell` `@carbon/styles`                                                                                                |
+| Popover                          | <Tag type="green">New</Tag>      | [SASS Modules](#sass-modules)                       | `Tooltip`                                                                                                                  |
+| Tooltip                          | <Tag type="cyan">Breaking</Tag>  | [SASS Modules](#sass-modules)                       | `Tooltip` `@carbon/react`                                                                                                  |
+| `carbon-components-react`        | <Tag type="red">Deprecated</Tag> | [SASS Modules](#sass-modules)                       | v11.x                                                                                                                      |
+| `carbon-icons`                   | <Tag type="red">Deprecated</Tag> | [SASS Modules](#sass-modules)                       | v11.x                                                                                                                      |
+| `carbon-components`              | <Tag type="red">Deprecated</Tag> | [SASS Modules](#sass-modules)                       | v11.x                                                                                                                      |
+
+## Components
+
+### Step 1: Install the new package
+
+```
+npm -i @carbon/react
+```
+
+or
+
+```
+yarn install @carbon/react
+```
+
+### Step 2: Remove the old packages
+
+```
+npm uninstall carbon-components carbon-components-react carbon-icons
+```
+
+or
+
+```
+yarn remove carbon-components carbon-components-react carbon-icons
+```
+
+something here about updating imports to point to `@carbon/react` instead of
+`carbon-components-react`, maybe explaing that the components all still work the
+same just coming from a diff package now. and why did we do this???
```

Comment:
The one-liner for why we did this was to provide a one-package install for using Carbon instead of having to install three separate versions and keep them in sync.